src/resources/clients.ts

```diff
@@ -17,15 +17,18 @@ import KeyStoreConfig from '../defs/keystoreConfig';
 import ResourceServerRepresentation from '../defs/resourceServerRepresentation';
 import ScopeRepresentation from '../defs/scopeRepresentation';
 
-export interface ClientQuery {
+export interface PaginatedQuery {
```

Comment:
Nice, it makes a lot of sense to split this out into a separate type. Perhaps it should be a level higher up though as a separate type from clients?